bulbs/special_coverage/migrations/0009_auto_20160208_1540.py

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.db import migrations, models
+from django.db import migrations
+
+from bulbs.special_coverage.models import SpecialCoverage
```

Comment:
Have you re-run this locally? 

Migrations can have dependency issues if not explicit, could be wrong here.